x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../ftr_provider_context';
+
+const FINDINGS_ROUTE = 'cloud_security_posture/findings';
```

Comment:
at some point we might want to extract some constants to `@kbn/cloud-security-posture-plugin-consts` or similar and use them in plugin/FTR to avoid duplication.